devops/README.md

```diff
@@ -62,8 +62,6 @@ Finally, a production deployment to all our servers would look like:
 
 ## Deploying Drupal websites
 
-For example, for installing just the `ac` Drupal project:
+The drupal playbook requires a `website` variable with the short name of the website to be updated.
```

Comment:
When a role requires a varable to be set, I sometimes do the following as the first task:

```yaml
- name: Check that all parameters are defined
  fail:
    msg: "{{ item }} needs to be defined"
  when: vars[item] is undefined
  loop:
    - flexextract_src_dir
    - flexextract_user_name
```